streamingpro-mlsql/src/main/java/streaming/dsl/mmlib/algs/SQLLDA.scala

```diff
@@ -27,36 +29,190 @@ class SQLLDA extends SQLAlg with Functions {
     emptyDataFrame()(df)
```

Comment:
We do not recommend to return a `emptyDataFrame` in train method. Can you try to use `formatOutput(getModelMetaData(spark, path))` ?  `formatOutput` will output some information e.g. path, is successful or not.People will know nothing if the training stage no exception is thrown